docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -43,90 +46,79 @@ $ cd tpcc
 The binaries are compiled with JAVA 13 and it is recommended to run these binaries with that version.
 {{< /note >}}
 
-## Step 2. Start your database
+### Step 2. Start your database
```

Comment:
This step is not really adding value. You can move this to prerequisites.

Reply:
Done.